main.cpp

```diff
@@ -0,0 +1,75 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <stdio.h>
+
+
+int main(int argc, char *argv[]) {
+    // std::ofsteam configfile;
+   // configfile.open("known_recipients.txt");
+
+    //std::string argument1 = "";
+
+//std::string user_input = "";
+
+//Location
+
+//if(argument = init)
+    std::string first_and_last ="", email = "", cypher = "", timezone, path ="";
+
+    std::cout << "Enter your first and last name, please" << std::endl;
+    std::getline(std::cin, first_and_last);
+
+        while(first_and_last == ""){
+        std::cout << "Please enter your first and last name to continue" << std::endl;
```

Comment:
Think flow charts here when you do a while loop with a variable condition what part of the code do you have to repeat to avoid having an infinite loop?

Because this could definitely cause an infinite loop